cmsl1t/__init__.py

```diff
@@ -22,9 +24,11 @@
 logger.addHandler(ch)
 
 if 'PROJECT_ROOT' not in os.environ:
-    logger.warn("Could not find environmental variable 'PROJECT_ROOT'")
-    logger.warn("You should to run 'source setup.sh' first!")
+    logger.warning("Could not find environmental variable 'PROJECT_ROOT'")
+    logger.warning("You should to run 'source setup.sh' first!")
     HERE = path.dirname(path.abspath(__file__))
     PROJECT_ROOT = path.abspath(path.join(HERE, path.pardir))
 else:
     PROJECT_ROOT = os.environ['PROJECT_ROOT']
+
+PY3 = sys.version_info[0] == 3
```

Comment:
Do you really need to define this?  It seems reasonable to me to let each piece of code that needs to know this either define it itself, or directly check `sys.version_info`.  If you do really want it, I'm not sure `PY3` is a great name.  `is_python_3` makes it more obvious that it's a boolean and I think would conform better with PEP8.

Reply:
I agree, it is not a good solution. I should use `six.PY3` instead :).